### PR TITLE
Chinese translation

### PR DESCRIPTION
**Pull request by J0**
## What kind of change does this PR introduce?

Baseline chinese translation for README.md. Would be nice to get a native speaker to review though!

## What is the current behavior?
NA

## What is the new behavior?

Basic Chinese translation.  Helps out with #1341 

## Additional context

Add any other context or screenshots.

### CODE REVIEW

**Comment by kiwicopple** [comment]
OK cool! Perhaps @w3b6x9 can review?

We also have `zh-tw` now, so perhaps we can rename this file to `zh-cn`?

**Comment by w3b6x9** [comment]
@kiwicopple

> OK cool! Perhaps @w3b6x9 can review?

Sorry, can't really help with this one! I can really only speak Chinese at this point in my life.

File: i18n/README.cn.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- [x] 实时订阅
+- [x] 身份验证和授权
+- [x] 自动生成的 api
+- [x] 指示板
+- [x] 存储
+- [ ] 函数(开发中)
+
+## 文档
+
+我们的文档，请到 [supabase.io/docs](supabase.io/docs)
```
Comment (J0) [inline i18n/README.cn.md:21]: whoops need to edit this

**Comment by songhn233** [comment]
@J0 Hi, i can help review the translation as a native speaker.

**Comment by w3b6x9** [comment]
@songhn233 thank you!

**Comment by J0** [comment]
Thanks @songhn233! :)

File: i18n/README.cn.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## 文档
+
+完整的说明文件请见[supabase.io/docs](supabase.io/docs)
+
+## 社群与支援
+
+社区论坛。最适合: 帮助构建和讨论数据库最佳实践。
+Github Issues: 电子邮件支持。最适合: 报告使用 Supabase 时遇到的问题。
+电子邮件支持。最适合:使用数据库或基础设施时遇到的问题。
```
Comment (songhn233) [inline i18n/README.cn.md:27]: I think 'Best for' should be translated as "适用于".

File: i18n/README.cn.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+- [ ] 函数(开发中)
+
+## 文档
+
+完整的说明文件请见[supabase.io/docs](supabase.io/docs)
+
+## 社群与支援
+
+社区论坛。最适合: 帮助构建和讨论数据库最佳实践。
+Github Issues: 电子邮件支持。最适合: 报告使用 Supabase 时遇到的问题。
```
Comment (songhn233) [inline i18n/README.cn.md:26]: "电子邮件支持" is the next line.

File: i18n/README.cn.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+电子邮件支持。最适合:使用数据库或基础设施时遇到的问题。
+
+## 发展现状
+
+- [x] Alpha: 我们正在对有限的一批客户进行 Supabase 测试
+- [x] 公开 Alpha: 任何人都可以在[app.supabase.io](https://app.supabase.io)上进行注册。
+- [x] 公开 Beta: 稳定,可以支持大多数非企业用例
+- [ ] 公开: 生产就绪软件
```
Comment (songhn233) [inline i18n/README.cn.md:34]: 'Public' can be left untranslated by convention.

File: i18n/README.cn.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+我们目前处于公开测试阶段。访问这个存储库的发布页面以获得关于主要更新的通知。
```
Comment (songhn233) [inline i18n/README.cn.md:36]: We are currently in Public Beta. Watch "releases" of this repo to get notified of major updates.

我们目前处于 Public Beta 阶段。关注 "Releases" 以获得关于重大更新的通知。

File: i18n/README.cn.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+<kbd><img src="https://gitcdn.link/repo/supabase/supabase/master/web/static/watch-repo.gif" alt="Watch this repo"/></kbd>
+
+---
+
+## 系统的运行
```
Comment (songhn233) [inline i18n/README.cn.md:42]: 系统的运行 -> 工作方式

File: i18n/README.cn.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+Supabase 是一个开源工具的组合。我们使用企业级的开源产品来构建 Firebase 的特性。如果工具和社区存在，并且有 MIT、Apache 2 或同等的开放许可，我们将使用并支持该工具。如果这个工具不存在，我们将自己开发和开源它。Supabase 不是 Firebase 的一对一映射。我们的目标是让开发人员使用开源工具获得类似 firebase 的开发体验。
```
Comment (songhn233) [inline i18n/README.cn.md:44]: 如果工具和社区存在 -> 如果存在相应的工具和社区

File: i18n/README.cn.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+**当前的体系结构**
```
Comment (songhn233) [inline i18n/README.cn.md:46]: 当前的体系结构 -> 当前架构

File: i18n/README.cn.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+Supabase 是一个[托管平台](https://app.supabase.io)。您可以注册并开始使用 Supabase，而无需安装任何软件。我们仍在创造本地开发体验——这是我们现在的核心焦点，以及平台的稳定性。
```
Comment (songhn233) [inline i18n/README.cn.md:48]: We are still creating the local development experience - this is now our core focus, along with platform stability.

我们仍在优化本地开发体验，这和平台稳定性一起作为我们当前的核心目标。

File: i18n/README.cn.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+
+![软件架构图](https://supabase.io/assets/images/supabase-architecture-9050a7317e9ec7efb7807f5194122e48.png)
+
+- [PostgreSQL](https://www.postgresql.org/)是一个对象关系数据库系统，经过了超过 30 年的积极开发，在可靠性、特性健壮性和性能方面赢得了声誉。
+- [Realtime](https://github.com/supabase/realtime)是一个 Elixir 服务器，允许你监听 PostgreSQL 插入，更新和删除使用 websockets。Supabase 监听 Postgres 内置的复制功能，将复制字节流转换成 JSON，然后通过 websocket 广播 JSON。
+- [PostgREST](http://postgrest.org/)是一个 web 服务器，把你的 PostgreSQL 数据库直接变成一个 RESTful API。
+- [Storage](https://github.com/supabase/storage-api)提供一个 RESTful 接口来管理存储在 S3 中的文件，使用 Postgres 管理权限。
+
+- [postgres-meta](https://github.com/supabase/postgres-meta) 是一个用于管理 Postgres 的 RESTful API，允许你获取表，添加角色，运行查询等。
+- [GoTrue](https://github.com/netlify/gotrue)这是一个基于 SWT 的 API，用于管理用户和发布 SWT 令牌 a。
```
Comment (songhn233) [inline i18n/README.cn.md:58]: 'a' is a typo?

File: i18n/README.cn.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+
+**当前的体系结构**
+
+Supabase 是一个[托管平台](https://app.supabase.io)。您可以注册并开始使用 Supabase，而无需安装任何软件。我们仍在创造本地开发体验——这是我们现在的核心焦点，以及平台的稳定性。
+
+![软件架构图](https://supabase.io/assets/images/supabase-architecture-9050a7317e9ec7efb7807f5194122e48.png)
+
+- [PostgreSQL](https://www.postgresql.org/)是一个对象关系数据库系统，经过了超过 30 年的积极开发，在可靠性、特性健壮性和性能方面赢得了声誉。
+- [Realtime](https://github.com/supabase/realtime)是一个 Elixir 服务器，允许你监听 PostgreSQL 插入，更新和删除使用 websockets。Supabase 监听 Postgres 内置的复制功能，将复制字节流转换成 JSON，然后通过 websocket 广播 JSON。
+- [PostgREST](http://postgrest.org/)是一个 web 服务器，把你的 PostgreSQL 数据库直接变成一个 RESTful API。
```
Comment (songhn233) [inline i18n/README.cn.md:54]: 直接变成一个 RESTful API -> 直接生成 RESTful API

**Comment by J0** [comment]
@songhn233  that was a really quick turnaround. Thanks for taking the time to do this!

**Comment by songhn233** [comment]
@J0 Thanks for your commits, the translation can now be merged, i think.

**Comment by kiwicopple** [comment]
thanks team 🔥

**Pull request merged**